build-files/kleborate/3.2.4-micromamba/Dockerfile

```diff
@@ -0,0 +1,117 @@
+
+FROM app AS test
+
+WORKDIR /test
+
+# from https://kleborate.readthedocs.io/en/latest/Installation.html
+RUN wget -q https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/002/813/595/GCF_002813595.1_ASM281359v1/GCF_002813595.1_ASM281359v1_genomic.fna.gz && \
+kleborate -a GCF_002813595.1_ASM281359v1_genomic.fna.gz -o kleborate_test -p kpsc && \
+head kleborate_test/klebsiella_pneumo_complex_output.txt
+
+# test e. coli specific modules
+# STEC genome from here: https://www.ncbi.nlm.nih.gov/datasets/genome/GCA_018509925.1/
+# expect to see stx2d_operon detected
+RUN wget -q https://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/018/509/925/GCA_018509925.1_PDT001047072.1/GCA_018509925.1_PDT001047072.1_genomic.fna.gz && \
+gunzip GCA_018509925.1_PDT001047072.1_genomic.fna.gz && \
+kleborate -a GCA_018509925.1_PDT001047072.1_genomic.fna -o kleborate_ecoli_test --preset escherichia && \
+grep 'stx2d' kleborate_ecoli_test/escherichia_output.txt
+
+# testing on another E. coli PNUSAE021029: https://www.ncbi.nlm.nih.gov/datasets/genome/GCA_012047485.2/
+# Known to have both stx2a and stx2c operons
+# Also known to be O157:H7 serotype
+# ectyper should report 'EHEC' pathotype
+# also known to have lots of virulence factors like eae, ehxA, many different esp genes (espA, espB, espJ, espK, espP, espX1)
+RUN wget -q https://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/012/047/485/GCA_012047485.2_PDT000417198.3/GCA_012047485.2_PDT000417198.3_genomic.fna.gz && \
+gunzip GCA_012047485.2_PDT000417198.3_genomic.fna.gz && \
+kleborate -a GCA_012047485.2_PDT000417198.3_genomic.fna -o kleborate-ecoli-test2 --preset escherichia && \
+grep 'stx2a' kleborate-ecoli-test2/escherichia_output.txt && \
+grep 'stx2c' kleborate-ecoli-test2/escherichia_output.txt && \
+grep 'O157:H7' kleborate-ecoli-test2/escherichia_output.txt && \
+grep 'EHEC' kleborate-ecoli-test2/escherichia_output.txt && \
+grep 'eae' kleborate-ecoli-test2/escherichia_output.txt 
+
+# testing on a particularly drug-resistant E. coli PNUSAE020982: https://www.ncbi.nlm.nih.gov/datasets/genome/GCA_012117185.1/
+# known to have blaCTX-M-27, sul1, tet(A), dfrA17, erm(B), gyrA_D87N, gyrA_S83L, parC_S80I
+RUN wget -q https://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/012/117/185/GCA_012117185.1_PDT000432447.1/GCA_012117185.1_PDT000432447.1_genomic.fna.gz && \
+gunzip GCA_012117185.1_PDT000432447.1_genomic.fna.gz && \
+kleborate -a GCA_012117185.1_PDT000432447.1_genomic.fna -o kleborate-ecoli-test3 --preset escherichia && \
+grep 'blaCTX-M-27' kleborate-ecoli-test3/escherichia_output.txt && \
+grep 'sul1' kleborate-ecoli-test3/escherichia_output.txt && \
+grep 'tet(A)' kleborate-ecoli-test3/escherichia_output.txt && \
+grep 'dfrA17' kleborate-ecoli-test3/escherichia_output.txt && \
+grep 'erm(B)' kleborate-ecoli-test3/escherichia_output.txt && \
+grep 'gyrA_D87N' kleborate-ecoli-test3/escherichia_output.txt && \
+grep 'gyrA_S83L' kleborate-ecoli-test3/escherichia_output.txt && \
+grep 'parC_S80I' kleborate-ecoli-test3/escherichia_output.txt
```

Comment:
Thank you for adding a test for E. coli! This will ensure that this image will work as intended.